lib/Mojolicious/resources/templates/mojo/debug.html.ep

```diff
@@ -17,9 +16,10 @@
     %= stylesheet '/mojo/mojo.css'
     <script>
       hljs.initHighlightingOnLoad();
-      $(function () {
-        $('[data-toggle="tooltip"]').tooltip()
-      });
+      const tooltipElements = document.querySelectorAll('[data-toggle="tooltip"]');
+      for (const tooltipElement of tooltipElements) {
```

Comment:
There is only a tooltip, I will remove the document.querySelectorAll, it was defensive programming too.